src/cookie-control-config.ts

```diff
@@ -77,6 +77,14 @@ export const cookieControlConfig: CookieControlConfig = {
 					event: "cookie.analytics.revoke",
 					cookieStorage: "none",
 				});
+
+				window.localStorage.removeItem("vwoSn");
+				for (let i = 0; i < window.localStorage.length; i++) {
+					const item = window.localStorage.key(i) || "";
```

Comment:
I'm not sure we need the `|| ""` - especially as we have the question mark on the next line

Reply:
Also, `item` isn't the best name - maybe `key` is better